zaza/charm_lifecycle/func_test_runner.py

```diff
@@ -81,14 +91,14 @@ def failure_report(model_aliases, show_juju_status=False):
                     default_flow_style=False))
 
 
-def run_env_deployment(env_deployment, keep_model=False, force=False,
+def run_env_deployment(env_deployment, keep_model=DESTROY_MODEL, force=False,
```

Comment:
The argument `keep_model` changing from a boolean to a 'enum' makes it a bit counterintuitive (to me), the argument maybe should be renamed to `strategy` to make the signature of the function more obvious (Just a nitpick, feel free to disregard you disagree)